lib/rollbar/js/middleware.rb

```diff
@@ -26,6 +26,7 @@ def call(env)
 
       def _call(env, result)
         return result unless should_add_js?(env, result[0], result[1])
+        @request = ::Rack::Request.new(env)
```

Comment:
I'd prefer not adding more state to this class when `#call` is executed. You can see I've even passed the `env` to the different methods as argument. I'd like to maintain that style.

So just passing this object to `config_js_tag` and `snippet_js_tag` and then to `script_tag` will be enough.